care/facility/migrations/0405_patientnotesedit.py

```diff
@@ -0,0 +1,84 @@
+# Generated by Django 4.2.2 on 2023-08-28 14:09
+
+import django.db.models.deletion
+from django.conf import settings
+from django.core.paginator import Paginator
+from django.db import migrations, models
+
+
+def create_initial_patient_notes_edit_record(apps, schema_editor):
+    PatientNotes = apps.get_model("facility", "PatientNotes")
+    PatientNotesEdit = apps.get_model("facility", "PatientNotesEdit")
+
+    notes_without_edits = (
+        PatientNotes.objects.annotate(
+            has_edits=models.Exists(
```

Comment:
You do not need to check the model which was just created, remove this filter/annotation.